bazelisk.go

```diff
@@ -396,11 +402,16 @@ func determineURL(fork string, version string, isCommit bool, filename string) s
 		kind = "rc" + versionComponents[1]
 	}
 
+	if baseURL != githubUpstream {
```

Comment:
It seems like we could simplify this:
- Remove the global constant `githubUpstream`.
- Remove the above check (L392 - 396).
- Move `baseURL := os.Getenv("BAZELISK_BASE_URL")` to above this line.
- Change this check to the above: `if len(baseURL) == 0`.
- Revert below to the old code.

WDYT?

Reply:
i did it a bit different to support commit based downloads too, plz check it out